ddtrace/llmobs/_integrations/base.py

```diff
@@ -34,9 +34,7 @@ def __init__(self, integration_config: IntegrationConfig) -> None:
 
     @property
     def span_linking_enabled(self) -> bool:
-        return asbool(os.getenv("_DD_LLMOBS_AUTO_SPAN_LINKING_ENABLED", "false")) or asbool(
-            os.getenv("_DD_TRACE_LANGGRAPH_ENABLED", "false")
-        )
+        return asbool(os.getenv("_DD_LLMOBS_AUTO_SPAN_LINKING_ENABLED", "false"))
```

Comment:
Using `os.getenv` makes it too hard to track down all accesses of a particular configuration. Use `ddtrace.settings._core.get_config` instead.

Reply:
thanks! turns out we already have this config set on the `config` object, if that's alright to use.

Reply:
this function has been removed entirely in a change in a different PR